python/build_images.py

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/python3.6
+
+import subprocess
+
+string=subprocess.check_output(["find", ".", "-name", "*.drawio", "-print"])
+arr=string.splitlines()
```

Comment:
I assume that `arr` is meant to be array? In Python this is a list, which of course is the name of a builtin function. Better to use more meaningful variables names like `found_files`.